helm-charts/templates/data-plane/gateway-components/gateway-runtime/listener-crt-secret.yaml

```diff
@@ -0,0 +1,19 @@
+  example.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURGRENDQWZ3Q0NRQ1U4c3RxZWk5OStUQU5CZ2txaGtpRzl3MEJBUXNGQURCTU1Rc3dDUVlEVlFRR0V3SlYKVXpFTE1Ba0dBMVVFQ0F3Q1EwRXhHREFXQmdOVkJBb01EMDE1UlhoaGJYQnNaU3dnU1c1akxqRVdNQlFHQTFVRQpBd3dOS2k1bGVHRnRjR3hsTG1OdmJUQWVGdzB5TXpBek1qZ3hNakV5TWpOYUZ3MHlOREF6TWpjeE1qRXlNak5hCk1Fd3hDekFKQmdOVkJBWVRBbFZUTVFzd0NRWURWUVFJREFKRFFURVlNQllHQTFVRUNnd1BUWGxGZUdGdGNHeGwKTENCSmJtTXVNUll3RkFZRFZRUUREQTBxTG1WNFlXMXdiR1V1WTI5dE1JSUJJakFOQmdrcWhraUc5dzBCQVFFRgpBQU9DQVE4QU1JSUJDZ0tDQVFFQTBFbjAybElkWGdQaWlZZ1UvVFBzL1FaNUZHWjhTUzREZjI3Qmh4ZmhpMFN6CjNRdkcrOVd0Q3pwK1duazFtbU4vWTFkY3pSOHRPNDlkTXdYOEFrQjRYRHFFK3hTQXF1Snp4RHV5QnhRTHZuWFQKYW8xbStJcHpsQVFrR011N1oxcUVQQmpYYUpERWtBb2ZCYUplc21CZlNuRm51ZGxPV2JCQ2IvcmRGU3NBSFRaSAprZ3FPTXE5TGk2cCsxczQ0bkhIWFptTEVOb1ZUdk8xZS9tWnhDVThrTDZCT0tHS0xlWnhJTWFBc0tOQnZ3NTZFCndleHVmUS9YSFJ0cG9CbndYUEJGbEhuZHdhNFhkcDJkVXovenZmWDIrbDhOUWxyRlY1SG10RlZUbWdnTnZCY04KT3YxcjduU2l5bEVCeDhxRlF2SHp4emRaMWQ0N2xUZnlCajZjQmwrQ2xRSURBUUFCTUEwR0NTcUdTSWIzRFFFQgpDd1VBQTRJQkFRQ2tCZGNxSlUvUnJTWkVJUGg3YnFMNVRnQ1ZvaDRoVHB5NnAwY2JEVVJNanVralpNSHRlWGNSClQ2UENFTkNvZDJHVHhLK2doMW5OTmFOWlJOc1VpYXVsVjQ3a0x0V05hUUV4QUE2US94UUNVK1ZLSmROVW1WVjkKMlhIM21kSE5OMWtPd0drNEM5ZTRzRnUyNGFSbkE2OEVsYm8wMmtpSmFxYThzVGNoRUtSRVhWUDVVYk42NHRrbQpuWXRyY2JDUFl6bUdGc2lFWVNMS01La0NzTDBpOHlqeWpCVlVjOWZWcm5TSXlmd0dBKy9nQlgrY3BVUFVpQlpJCkN2SWxhRDZBcG1PTWx1MmZLUW1DOWttdExPc29SREcrSS9LNVdxa3VWemQ4UGU4cWZRQW5tSE1wTXVSczB4REMKcVM1ZTYwOTNadUtyMlZMRjhVR3A0dWNYZ0hRbVJwcXgKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQ==
```

Comment:
shall we remove this unused cert/key pair

Reply:
it's used in the example listener which we have added in the gateway cr, anyway mounting secrets approach will be changed in  the next iteration to read from cert refs